### DISCUSSION TITLE
Projects that use this extension

### DISCUSSION THREAD

**Discussion by melissawm** [melissawm/sphinx-tags#32] [Announcements]
Hi folks! 

In an effort to better understand the demand for improvements, please add a note below with the name and repo for projects that are using sphinx-tags for documentation.

Cheers!

**Comment by melissawm** [comment]
sphinx-tags is used for the gallery examples in [napari.org](https://napari.org/dev/gallery.html)

Repo: https://github.com/napari/docs

Usage: https://github.com/napari/docs/blob/48443c792b466dff0a47b22317c2e58ce3eba04f/docs/conf.py#L66

**Comment by mikofski** [comment]
Hi @melissawm , [DISCOVER Cookbook is testing out sphinx-tags in PR60](https://github.com/numfocus/DISCOVER-Cookbook/pull/60)

**Reply by melissawm** [reply]
Let me know if you need help setting it up!

**Reply by mikofski** [reply]
Sorry if this is in the docs, but can I use tags above any section label, or is it only per page? Also, Sphinx-Design didn't work easily for me in JBook, but it's no big deal. Thanks!

**Reply by melissawm** [reply]
It is only per-page now, but I will create an issue for that just in case. Thanks!

**Comment by AlexTheLion123** [comment]
[Trading Strategy docs](https://tradingstrategy.ai/docs/programming/strategy-examples/index.html#notebook-overview) are currently using sphinx-tags directly in their jupyter notebooks

**Comment by JWCook** [comment]
I'm using this for a small personal website / possible future blog. There's not much content, just playing around to get a feel for how well Sphinx+MyST is suited for making this kind of site.

Site: https://jwcook.tilde.team
Source: https://tildegit.org/jwcook/jwcook.tilde.team (sphinx-tags config [here](https://tildegit.org/jwcook/jwcook.tilde.team/src/commit/9aa2f57370f0ad1803e4931d70f0f5cd4ae33b8a/pages/conf.py#L41-L53))

**Reply by melissawm** [reply]
Let me know if you have suggestions or feature requests! Cheers 😄

**Comment by speth** [comment]
I am testing `sphinx-tags` for use in an update to Cantera's [example gallery](https://cantera.org/examples/python/index.html), as we transition to using `sphinx-gallery`. Currently, I have a PR open here: https://github.com/Cantera/cantera/pull/1621.

I found this via some discussion on the `sphinx-gallery` repo, and was very excited. I had gone through the process of tagging all of our examples a couple of years ago, with the idea of eventually using them for navigation, but never got around to writing something myself.

**Reply by melissawm** [reply]
Awesome, thanks for the heads up! 🎉

**Comment by DimitarGGeorgiev** [comment]
Hello,

I install extension the way described in:

https://sphinx-tags.readthedocs.io/en/latest/quickstart.html#installation

When I perform build, I recive error:

[
![build](https://github.com/melissawm/sphinx-tags/assets/90313714/3c6eb16d-d390-4389-907e-bebad5ce6bf1)
![extensions](https://github.com/melissawm/sphinx-tags/assets/90313714/4c466b97-d032-45dc-9175-29420530e38d)
![tag-install](https://github.com/melissawm/sphinx-tags/assets/90313714/844f2b4b-db77-42e0-8a85-7e0284ab4843)
![error-message](https://github.com/melissawm/sphinx-tags/assets/90313714/8b3fa138-ad48-43d5-95ef-d57016771744)
![list](https://github.com/melissawm/sphinx-tags/assets/90313714/00a00270-4c0f-47dc-8cb3-fbb2f7e437e2)

](url)

If I comment extension, build is complete.


Thank you in advance!

**Reply by melissawm** [reply]
Hi - this is probably best moved to a support topic, but did you add sphinx-tags as a requirement on your requirements.txt or environment.yml file?

**Reply by DimitarGGeorgiev** [reply]
Thank you for advance. I am new in sphinx language programming. Please, can you tell me how is correct to add extension: "sphinx-tags" or "sphinx_tags"?

**Reply by DimitarGGeorgiev** [reply]
Hello again.

I add sphinx-tags as a requirement in requirements.txt and now build is complete!

Thanks a lot for support!

Regards,
Dimitar

**Comment by fervand1** [comment]
Hi @melissawm thanks for having this extension. I am trying use sphinx tags in my documentation and It works fine. 

But I had a issue with sphinx-autobuild which I raised but not sure if it is a known issue or there is a config to overcome it, couldn't find it in the documentation.
Also is it possible to have tags as batches but with links? or it will be feature request? I was looking at sphinx panels here https://sphinx-panels.readthedocs.io/en/latest/ and something like linked badges could be a interesting feature to have. Thanks again

**Reply by fervand1** [reply]
Ignore my comment about hyperlinked badges, I see it working now but the the autobuild issue remains
![image](https://github.com/melissawm/sphinx-tags/assets/44014460/989cf666-43fd-4771-8f3f-6d6e4649c054)

**Comment by bjlittle** [comment]
Hey @melissawm,

Thanks for `sphinx-tags` it's really great!

Just thought that I'd let you know that I've started to use `sphinx-tags` over on [geovista](https://github.com/bjlittle/geovista) 🥳 

Lookin' forward to the proposed issue enhancements for the next release.

Cheers! 🍻

**Comment by RemDelaporteMathurin** [comment]
Thanks for this nice extension! @rekomodo and I are using on [FESTIM](https://github.com/festim-dev/FESTIM) 🚀 

We made an online [Verification & Validation book](https://festim-vv-report.readthedocs.io/) 📖

**Comment by tkoyama010** [comment]
Hi @melissawm,

Thanks for this awesome sphinx extention!

I'd let you know that We've started to use `sphinx-tags` over on [pyvista](https://github.com/pyvista/pyvista) 🚀

https://github.com/pyvista/pyvista/pull/6375

Special thanks for @adeak for review and @story645 for ideas
